### ISSUE TITLE
Is this compatible with the the flick gesture board by pi?

### ISSUE THREAD

**Issue by milajo2** [joshnishikawa/Flicker#2]
Hi, 
Is this compatible with the flick gesture board. 
https://uk.pi-supply.com/products/flick-large-standalone-3d-tracking-gesture-breakout

**Comment by joshnishikawa** [comment]
No, this library is for interpreting touch readings on individual pins of a compatible microcontroller (only Teensy's have been tested). The Flick board has its own dedicated chip reading something like 5 sensors and transmitting the data to the Pi or other controller via i2C (a completely different protocol).
